worker_tasks/scraper.py

```diff
@@ -67,7 +65,19 @@ def _store_items_and_category_counts(feed, categorizer):
 
         db.session.add(new_entry)
         db.session.commit()
-
+  
+    try:
+        feed.image_url = fp.feed.image.url
```

Comment:
I think we should only do this when we add a new feed in the api, or at least have a check to see if ```!feed.image_url``` first.